app/src/main/java/nl/avans/drivioapp/view/UpdateUFragment.kt

```diff
@@ -0,0 +1,111 @@
+package nl.avans.drivioapp.view
+
+import android.app.AlertDialog
+import android.os.Bundle
+import android.view.*
+import androidx.fragment.app.Fragment
+import android.widget.Toast
+import androidx.core.view.MenuHost
+import androidx.core.view.MenuProvider
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import nl.avans.drivioapp.R
+import nl.avans.drivioapp.databinding.FragmentUpdateUBinding
+import nl.avans.drivioapp.model.User
+import nl.avans.drivioapp.viewModel.UserViewModel
+
+class UpdateUFragment : Fragment() {
```

Comment:
Kan dit bestand niet beter UpdateUserFragment genoemd worden? De naam is nu namelijk niet heel veel zeggend..

Reply:
Aangepast in laatste commit